code/modules/holodeck/HolodeckControl.dm

```diff
@@ -87,6 +90,12 @@
 	else
 		dat += "Gravity is <A href='?src=\ref[src];gravity=1'><font color=blue>(OFF)</font></A><BR>"
 
+	if(!locked)
+		dat += "Holodeck is <A href='?src=\ref[src];togglehololock=1'><font color=green>(UNLOCKED)</font></A><BR>"
+	else
+		dat = "<B>Holodeck Control System</B><BR>"
```

Comment:
You are missing that header (Holodeck Control System) in the if-section.
Should probs move it to line 92 (so its always there regardless of the holodeck being locked or unlocked)

Reply:
Actually. Thats hacky but it should work. Its a = and not a +=.